src/genie/libs/parser/iosxe/tests/test_show_interface.py

```diff
@@ -524,7 +524,7 @@ class TestShowInterfacesSwitchport(unittest.TestCase):
             'pruning_vlans': '2-1001',
             'access_vlan': '1',
             'unknown_multicast_blocked': False,
-            'trunk_vlans': '1,111,130,131,400,405,410,420,430,439-442,450,451,460,',
+            'trunk_vlans': '1,111,130,131,400,405,410,420,430,439-442,450,451,460,470,480,490,500,616,619,700,709-712,720,723-725,760',
```

Comment:
Please run unit test and attach screen shot.

Reply:
![image](https://user-images.githubusercontent.com/19796896/111235860-d578ac00-85f1-11eb-9523-d8fab68f5524.png)